eng/versioning/version_data.txt

```diff
@@ -45,3 +45,4 @@ com.microsoft.azure.msi_auth_token_provider:azure-authentication-msi-token-provi
 com.microsoft.azure:azure-eventgrid;1.4.0-beta.1;1.4.0-beta.1
 com.microsoft.azure:azure-loganalytics;1.0.0-beta-2;1.0.0-beta.2
 com.microsoft.azure:azure-media;1.0.0-beta.1;1.0.0-beta.1
+com.microsoft.azure:azure-search;1.0.0-beta.2;1.0.0-beta.2
```

Comment:
Why was there an entry in version_data.txt for search? From my understanding azure-search is a client track library, not a data track library which menas there is no com.microsoft.azure:azure-search, only com.azure:azure-search, please remove this bogus entry.

Reply:
removed